### PR TITLE
BZ1923772: clarifying must-gather image stream for disconnected

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1923772

4.5+

new post-installation section: https://deploy-preview-34002--osdocs.netlify.app/openshift-enterprise/latest/post_installation_configuration/cluster-tasks.html#post-install-must-gather-disconnected
updated mirror section: https://deploy-preview-34002--osdocs.netlify.app/openshift-enterprise/latest/installing/installing-mirroring-installation-images.html#installing-preparing-samples-operator

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@bmcelvee, @rolfedh, @gabemontero, FYI

File: installing/installing-mirroring-installation-images.adoc [hunk 1]
```diff
@@ -62,9 +62,10 @@ In production environments, add the required images to a registry in your restri
 
 include::modules/installation-mirror-repository.adoc[leveloffset=+1]
 
-include::modules/installation-preparing-restricted-cluster-to-gather-support-data.adoc[leveloffset=+1]
+[id="installing-preparing-samples-operator"]
+== The Cluster Samples Operator in a disconnected environment
 
-include::modules/installation-restricted-network-samples.adoc[leveloffset=+1]
+In a disconnected environment, you must take additional steps after you install a cluster to configure Cluster Samples Operator. Review the following information in preparation.
```
Comment (mburke5678) [inline installing/installing-mirroring-installation-images.adoc:68]: s/Cluster Samples Operator/the Cluster Samples Operator

File: modules/installation-preparing-restricted-cluster-to-gather-support-data.adoc [hunk 2]
```diff
@@ -9,7 +9,23 @@ Clusters using a restricted network must import the default must-gather image to
 
 .Procedure
 
-* Import the default must-gather image from your installation payload:
+. If you have not added your mirror registry's trusted CA to your cluster's image configuration object as part of the Samples Operator configuration, do so:
```
Comment (mburke5678) [inline modules/installation-preparing-restricted-cluster-to-gather-support-data.adoc:12]: _do so_ seems a bit terse to me. _do the following_, _perform the following steps_?

File: modules/installation-preparing-restricted-cluster-to-gather-support-data.adoc [hunk 3]
```diff
@@ -9,7 +9,23 @@ Clusters using a restricted network must import the default must-gather image to
...
+.. Create the cluster’s image configuration object:
```
Comment (mburke5678) [inline modules/installation-preparing-restricted-cluster-to-gather-support-data.adoc:13]: I don't see this phrase elsewhere in the docs: _cluster's image configuration object_. 
I see _image configuration_. Should this be _image configuration custom resource (CR)_ or _image.config.openshift.io/cluster custom resource (CR)_, the latter [used here](https://docs.openshift.com/container-platform/4.7/openshift_images/image-configuration.html)?

File: modules/installation-preparing-restricted-cluster-to-gather-support-data.adoc [hunk 4]
```diff
@@ -9,7 +9,23 @@ Clusters using a restricted network must import the default must-gather image to
...
++
+[source,terminal]
+----
+$ oc create configmap registry-config --from-file=${MIRROR_ADDR_HOSTNAME}..5000=$path/ca.crt -n openshift-config
+----
+
+.. Add the required trusted CAs for the mirror in the cluster’s image
```
Comment (mburke5678) [inline modules/installation-preparing-restricted-cluster-to-gather-support-data.adoc:20]: Same as above.

File: modules/installation-preparing-restricted-cluster-to-gather-support-data.adoc (hunk 3 above)
Comment (kalexand-rh) [reply modules/installation-preparing-restricted-cluster-to-gather-support-data.adoc:13]: https://github.com/openshift/openshift-docs/blame/master/modules/installation-restricted-network-samples.adoc#L80

File: modules/installation-preparing-restricted-cluster-to-gather-support-data.adoc [hunk 5]
```diff
@@ -9,7 +9,23 @@ Clusters using a restricted network must import the default must-gather image to
 
 .Procedure
 
-* Import the default must-gather image from your installation payload:
+. If you have not added your mirror registry's trusted CA to your cluster's image configuration object as part of the Cluster Samples Operator configuration, perform the following steps:
```
Comment (mburke5678) [inline modules/installation-preparing-restricted-cluster-to-gather-support-data.adoc:12]: I'm not seeing this module in the preview(!).

Comment (mburke5678) [inline modules/installation-preparing-restricted-cluster-to-gather-support-data.adoc:12]: Does _ installing/install_config/installing-restricted-networks-preparations.adoc_ need to be updated to _post_installation_configuration/cluster-tasks_?

File: modules/installation-restricted-network-samples.adoc [hunk 6]
```diff
@@ -3,7 +3,11 @@
 // * installing/install_config/installing-restricted-networks-preparations.adoc
 // * openshift_images/samples-operator-alt-registry.adoc
 
-ifeval::["{context}" == "installing-restricted-networks-preparations"]
+ifeval::["{context}" == "installing-mirroring-installation-images"]
```
Comment (mburke5678) [inline modules/installation-restricted-network-samples.adoc:6]: Does _ installing/install_config/installing-restricted-networks-preparations.adoc_ need to be updated to _post_installation_configuration/cluster-tasks_?

**Comment by mburke5678** [comment]
@kalexand-rh A few nits. Otherwise LGTM.

**Pull request merged**